datalake-spark3/src/main/scala/bio/ferlab/datalake/spark3/genomics/prepared/VariantCentric.scala

```diff
@@ -85,11 +85,17 @@ import java.time.LocalDateTime
  *    }
  * }}}
  * @param rc the etl context
+ * @param destinationDataSetId custom destination dataset id
+ * @param enrichVariantsDataSetId custom enrich_variants dataset id
+ * @param enrichConsequencesDataSetId custom enrich_consequences dataset id
  */
-case class VariantCentric(rc: RuntimeETLContext) extends SimpleSingleETL(rc) {
-  override val mainDestination: DatasetConf = conf.getDataset("es_index_variant_centric")
-  private val enriched_variants: DatasetConf = conf.getDataset("enriched_variants")
-  private val enriched_consequences: DatasetConf = conf.getDataset("enriched_consequences")
+case class VariantCentric(rc: RuntimeETLContext,
```

Comment:
Not sure if I should also update de Object associated with the case class ?

Reply:
Faut modifier le run